x-pack/plugins/security_solution/public/common/components/visualization_actions/lens_attributes/common/alerts/alerts_table.ts

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { v4 as uuidv4 } from 'uuid';
+import type { GetLensAttributes } from '../../../types';
+
+const layerId = uuidv4();
+
+export const getAlertsTableLensAttributes: GetLensAttributes = (
+  stackByField = 'kibana.alert.rule.name',
+  extraOptions
+) => {
+  return {
+    title: 'Alerts',
+    description: '',
+    visualizationType: 'lnsDatatable',
+    state: {
+      visualization: {
+        columns: [
+          {
+            columnId: '2881fedd-54b7-42ba-8c97-5175dec86166',
+            isTransposed: false,
+            width: 362,
+          },
+          {
+            columnId: 'f04a71a3-399f-4d32-9efc-8a005e989991',
+            isTransposed: false,
+          },
+          {
+            columnId: '75ce269b-ee9c-4c7d-a14e-9226ba0fe059',
+            isTransposed: false,
```

Comment:
This column shouldn't have actions because it is a count. It is currently filtering by `host.name: 3971`
<img width="1496" alt="Screenshot 2023-02-01 at 13 57 08" src="https://user-images.githubusercontent.com/1490444/216050291-4e3dc188-52a2-4055-86c2-58a0abb9f0aa.png">

Reply:
https://github.com/elastic/kibana/issues/150048

Reply:
seems that they add agent.type:* when filtering, so I find it alright. However our action Open in timeline and copy to clip board has the wrong value as it added agent.type: "3971" , this is definitely a bug.

so I guess we’ll just fix the Open in timeline and copy to clipboard actions we added and keep the cell actions.